pkg/common/config/eventingnatsconfig.go

```diff
@@ -60,3 +65,12 @@ type ENConfigRootCA struct {
 	// Secret is a reference to an existing Secret where the controller will extract a certificate by the "ca.crt" key.
 	Secret *v1.LocalObjectReference `json:"secret,omitempty"`
 }
+
+type ConnOpts struct {
+	// MaxReconnects how many attempts to reconnect
+	MaxReconnects int `json:"maxReconnects,omitempty"`
+	// RetryOnFailedConnect should retry on failed reconnect
+	RetryOnFailedConnect bool `json:"retryOnFailedConnect,omitempty"`
+	// ReconnectWait time between reconnects in milliseconds
+	ReconnectWait time.Duration `json:"reconnectWait,omitempty"`
```

Comment:
Usually, Kubernetes recommends to have the unit in the name: https://github.com/zecke/Kubernetes/blob/master/docs/devel/api-conventions.md#units and the type being just `int` the reason being, time.Duration is a Go only type and format, so it limits the client that can easily use the API

Reply:
fixed